app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("Visitor isn't vaccinated!")
+
+        if visitor.get("vaccine").get("expiration_date") \
+                < \
+                datetime.date.today():

```

Comment:
So bad readability.... Also .get for vaccine is not needed. And for expiration date also, because if vaccine present we always have expiration_date
```suggestion
        expiration_date = visitor["vaccine"]["expiration_date"]
        if expiration_date < datetime.date.today():
```